gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -143,15 +189,21 @@ protected void updateExpiration(final String tokenId, long expiration) {
 
   @Override
   protected List<String> getTokens() {
-    List<String> allAliases = new ArrayList<>();
+    List<String> tokenIds = null;
+
     try {
-      allAliases = aliasService.getAliasesForCluster(AliasService.NO_CLUSTER_NAME);
-      /* only get the aliases that represent tokens and extract the current list of tokens */
-      allAliases = allAliases.stream().filter(a -> a.contains(TOKEN_MAX_LIFETIME_POSTFIX)).map(a -> a.substring(0, a.indexOf(TOKEN_MAX_LIFETIME_POSTFIX)))
-          .collect(Collectors.toList());
+      List<String> allAliases = aliasService.getAliasesForCluster(AliasService.NO_CLUSTER_NAME);
+
+      // Filter for the token state aliases, and extract the token ID
+      tokenIds = allAliases.stream()
+                           .filter(a -> a.contains(TOKEN_MAX_LIFETIME_POSTFIX))
+                           .map(a -> a.substring(0, a.indexOf(TOKEN_MAX_LIFETIME_POSTFIX)))
+                           .collect(Collectors.toList());
     } catch (AliasServiceException e) {
-      log.errorEvictingTokens(e);
+      e.printStackTrace(); // TODO: PJZ: Logging
```

Comment:
Why are we throwing a stack trace? if this is for debugging can we wrap it up in logger and then throw instead of using e.printStackTrace().

Reply:
This is an oversight. Thanks for catching it.